### PR TITLE
chore(.travis): check spec for errors/warnings

### PR DESCRIPTION
**Pull request by marcoscaceres**
Part of #733 

This change (choose one):

* [x] Is a "chore" (metadata, formatting, fixing warnings, etc).

Commit message:

Check the spec on Travis, meaning we can drop the need to manually check for the spec.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/pull/739.html" title="Last updated on Nov 13, 2018, 6:02 AM GMT (6abed2f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/739/16c5d13...6abed2f.html" title="Last updated on Nov 13, 2018, 6:02 AM GMT (6abed2f)">Diff</a>

### CODE REVIEW

File: tools/validate.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+"use strict";
+const { exec } = require("child_process");
+const handler = require("serve-handler");
+const http = require("http");
+
+/**
+ *
+ * @param {string} cmd A string representing a shell command.
+ */
+function toExecutable(cmd) {
```
Comment (mgiuca) [inline tools/validate.js]: This name is a bit confusing. ("It's generating an executable file?")

How about CommandRunner? Thus, `CommandRunner(cmd).run()`.

Comment (marcoscaceres) [reply tools/validate.js]: Good suggestion. Will change.

Comment (marcoscaceres) [reply tools/validate.js]: Will just make it a class called `ShellCommand`.

**Comment by marcoscaceres** [comment]
> FYI, I don't know anything about Node or Travis. It looks correct, but I'm not an expert.

That's ok :) Feedback was useful. I verified it fails and passes - so should be good.

**Pull request merged**